end_to_end/music_recommendation/02_export_feature_groups.ipynb

```diff
@@ -0,0 +1,1151 @@
+{
```

Comment:
<p>Grammar: <code>check if the athena queres</code> -&gt; <code>Check if the Athena queries</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3392/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Reply:
<p>Fixed</p>